src/pages/index.js

```diff
@@ -163,8 +272,24 @@ editModalCloseBtn.addEventListener("click", () => {
 
 cardModalBtn.addEventListener("click", () => {
   openModal(cardModal);
+  disableButton();
```

Comment:
`  disableButton();`

Please, delete it

You need to **disable** the button only in a `.then` block after a **successful response** to be able to click it again if there is a server error.